### DISCUSSION TITLE
Customizing the Side Editor Interface and Position

### DISCUSSION THREAD

**Discussion by Rafath-Auvee** [pdfme/pdfme#183] [Q&A]
Issue Description: I would like to customize the side editor interface in the PDFme to enhance its visual appeal and improve user experience. Currently, the editor has a white background, which I would like to change. Additionally, I would like to move the side editor to either the navbar or the top position.

My question is, how can I achieve these customizations in the PDFme? Can I accomplish this using CSS styling? I would appreciate guidance on the necessary steps or code modifications needed to implement these changes effectively.

Expected Outcome:

Ability to customize the background color of the side editor.

Ability to reposition the side editor to the navbar or the top position.

Clear instructions or code examples on how to apply CSS styling for these customizations.

Thank you for your assistance!

**Comment by yusufayhan0** [comment]
> Issue Description: I would like to customize the side editor interface in the PDFme to enhance its visual appeal and improve user experience. Currently, the editor has a white background, which I would like to change. Additionally, I would like to move the side editor to either the navbar or the top position.
> 
> My question is, how can I achieve these customizations in the PDFme? Can I accomplish this using CSS styling? I would appreciate guidance on the necessary steps or code modifications needed to implement these changes effectively.
> 
> Expected Outcome:
> 
> Ability to customize the background color of the side editor.
> 
> Ability to reposition the side editor to the navbar or the top position.
> 
> Clear instructions or code examples on how to apply CSS styling for these customizations.
> 
> Thank you for your assistance!

I am waiting for an answer to this question

**Comment by abdullahonden** [comment]
> Issue Description: I would like to customize the side editor interface in the PDFme to enhance its visual appeal and improve user experience. Currently, the editor has a white background, which I would like to change. Additionally, I would like to move the side editor to either the navbar or the top position.
> 
> My question is, how can I achieve these customizations in the PDFme? Can I accomplish this using CSS styling? I would appreciate guidance on the necessary steps or code modifications needed to implement these changes effectively.
> 
> Expected Outcome:
> 
> Ability to customize the background color of the side editor.
> 
> Ability to reposition the side editor to the navbar or the top position.
> 
> Clear instructions or code examples on how to apply CSS styling for these customizations.
> 
> Thank you for your assistance!

I'm working on a project and I want to know if there is such a feature.

**Comment by peteward** [comment]
Hi, 

There's no ability to do this currently - there is not much customisation of the UI possible.
CSS control is not easy either because the HTML mark-up has nothing of use to identify different components - changing this (adding IDs or classes to components)  might be a first step to helping you achieve some flexibility without forking the repo.
